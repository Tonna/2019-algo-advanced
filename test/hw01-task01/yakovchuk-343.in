25
1 1 1 3 4 1 4 5 6 1 4 4 2 12 2 4 6 10 2 8 9 17 15 15
9
25 7
16 8
11 9
2 9
16 3
12 8
2 2
5 5
8 2
