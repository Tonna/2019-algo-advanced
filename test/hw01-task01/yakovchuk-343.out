23
15
26
18
5
16
4
9
5
